index.html

```diff
@@ -6,27 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" type="text/css" href="style.css" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500;700&display=swap">
-    <title>Portfolio page </title>
+    <title>My Portfolio page </title>
 </head>
 <body>
     <div class="main-one">
         <header>
             <h1 class="mylogo">Ndonyi</h1>
-            <menu>
-                <ul class="home">
+            
+                <ul id="home">
                     <li class="content"><a href="#">Portoflio</a></li>
-                    <li class="content"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
-                    <li class="content"><a href="https://citec-47.github.io/Desk-version/">Contact</a></li>
+                    <li class="content"><a href="#">About</a></li>
+                    <li class="content"><a href="#">Contact</a></li>
+                </ul>
```

Comment:
- [ ] As noted by the previous reviewers, these links should lead directly to the section of the page. for example, look for the `id` or `class` of the `About` section, then insert it into the `<a href="#(id here)">`. this will make the three links link to a part of the project when clicked upon. Kindly make this change 👍